library/include/hipblas.h

```diff
@@ -16693,7 +16693,11 @@ HIPBLAS_EXPORT hipblasStatus_t hipblasZgetriBatched(hipblasHandle_t
     ldb         int. ldb >= max(m,n).\n
                 Specifies the leading dimension of matrix B.
     @param[out]
-    info        pointer to int on the GPU.\n
+    info      pointer to a int on the host.\n
+              If info = 0, successful exit.
+              If info = j < 0, the j-th argument is invalid.
```

Comment:
Very minor, but it looks like the parameter description is misaligned with other parameters.